src/ert/gui/main.py

```diff
@@ -96,6 +96,10 @@ def _start_initial_gui_window(
             # Changing current working directory means we need to update
             # the config file to be the base name of the original config
             args.config = os.path.basename(args.config)
+
+            if plugin_manager:
+                ErtConfig.add_preinstalled_fm_steps(plugin_manager.forward_model_steps)
```

Comment:
The main function is never called with a plugin_manager in tests, that is line 101 is not covered.

Reply:
Added integration test to cover a cli run where there is a plugin manager with a plugin fm step